test/signs_ui_web/channels/signs_channel_test.exs

```diff
@@ -0,0 +1,89 @@
+defmodule SignsUiWeb.SignsChannelTest do
+  use SignsUiWeb.ChannelCase
+  import ExUnit.CaptureLog
+
+  setup do
+    socket = subscribe_and_join!(socket(), SignsUiWeb.SignsChannel, "signs:all", %{})
+    %{socket: socket}
+  end
+
+  describe "handle_in" do
```

Comment:
Could some of these use `@tag :authenticated`?

Reply:
The `@tag :authenticated` thing is implemented for conn case but not for channel case. We had a lot of trouble with implementing a setup block for channel cases that allows us to do a `@tag :authenticaed`-type thing, so these tests just use a combination of `encode_and_sign` and `assign_rts` if they need an authenticated socket.